src/MallardMessageHandlers/AuthenticationToken/AuthenticationTokenHandler.cs

```diff
@@ -11,8 +11,10 @@ namespace MallardMessageHandlers
 {
 	/// <summary>
 	/// This <see cref="HttpMessageHandler"/> handles the access token logic.
-	/// If the request should include an authentication token <see cref="ShouldIncludeToken(HttpRequestMessage)"/>,
-	/// the handler gets the token from the <see cref="IAuthenticationTokenProvider{TAuthenticationToken}"/>.
+	/// By default, a token is included in every request, except if the request already contains an
+	/// authorization header with its scheme set to "Anonymous".
+	/// This behavior can be modified by overriding the <see cref="ShouldIncludeToken(HttpRequestMessage)"/>.
+	/// By default, the authorization header scheme is "Bearer", but it can be overriden in a derived class.
```

Comment:
Wouldn't it make sense to make the setter public instead of forcing inheritance?

Reply:
By forcing inheritance, the value can be calculated based on the call that's being made, which would not be possible with a simple setter. I can make the setter public, but I doubt that this brings any kind of flexibility : why would one add an AuthenticationTokenHandler to a request pipeline if they're going to configure it to NOT include the authentication header?